x/ibc/version.go

```diff
@@ -0,0 +1,3 @@
+package ibc
+
+const Version int64 = 1
```

Comment:
Where is this used?

Reply:
For version prefixes(ibc v1  can be stored under prefix "v1/", ...). Can be deleted, we can simply use multiple IAVLStores per version